Code/Mantid/MantidQt/MantidWidgets/inc/MantidQtMantidWidgets/PeriodicTableWidget.h

```diff
@@ -0,0 +1,61 @@
+#ifndef MANTID_MANTIDWIDGETS_PERIODICTABLE_H_
+#define MANTID_MATIDWIDGETS_PERIODICTABLE_H_
```

Comment:
line 2: there's a typo. Clang luckily realized and is giving a warning about that.